client/extensions/woocommerce/components/form-dimensions-input/style.scss

```diff
@@ -0,0 +1,45 @@
+.form-dimensions-input {
+	display: flex;
+	justify-content: flex-start;
+	flex-wrap: wrap;
+	@include breakpoint( ">480px" ) {
```

Comment:
The guidelines actually don't mention of single vs double quotes, but the examples show double quotes. https://github.com/Automattic/wp-calypso/blob/de700a238a201232c8481b59b4f7feba7700e977/docs/coding-guidelines/css.md#media-queries